README.adoc

```diff
@@ -77,7 +79,7 @@ The command `search` will search for all available packages. It should then show
 
 [source,bash,options="nowrap"]
 ----
-skipper:>search
+skipper:>package search
```

Comment:
We could also update the statement above as, The command `package search` will search...

Reply:
good catch, thanks & updated!